release/Makefile

```diff
@@ -4,12 +4,18 @@ PACKAGE_NAME = github.com/projectcalico/calico/release
 
 include ../lib.Makefile
 
+# The name of the product.
+# Either "Calico" or "Calico Enterprise"
+PRODUCT_NAME=Calico
+
 export ORGANIZATION
 export GIT_REPO
 export GIT_REPO_SLUG
 export DEV_REGISTRIES
 export OPERATOR_BRANCH
 
+LDFLAGS= -X $(PACKAGE_NAME)/pkg/buildinfo.ProductName='$(PRODUCT_NAME)'
```

Comment:
Why do we need to pass this in via LDFLAGs? What is the benefit of this over defining it as a constant in Golang?